data/transport missions.txt

```diff
@@ -1678,6 +1676,22 @@ mission "Shady passenger transport 3 - double cross"
 
 
 
+mission "Shady passenger transport 3 - double cross big payment"
+	name "Shady passenger transport big payment"
+	landing
+	invisible
+
+	to offer
+		has "bribed mercenaries"
```

Comment:
Just in case, this should still have `credits >= 8000000` as a `to offer` condition; the chance might be slight now, but it's still possible to not have enough credits for the negative payment here when the player did have enough when the original mission was offered.